requirements-dev.txt

```diff
@@ -7,3 +7,4 @@ seed-isort-config~=2.1
 sphinx~=3.1
 sphinx_rtd_theme~=0.4
 tox~=3.14
+pytest~=5.2
```

Comment:
We may want this to be `>=` instead.

Reply:
`>=` is better than `~=` for compatibility with other packages, but it's worse for the stability of gt4py, since new major versions of pytest could change the API in a non backward compatible way. In this case, since `pytest` is not critical for the gt4py internals, I think it could be worth to change it as you suggest.